res/pokemon/meson.build

```diff
@@ -0,0 +1,534 @@
+subdir('baltoy')
+subdir('claydol')
+subdir('lileep')
+subdir('cradily')
+subdir('anorith')
+subdir('armaldo')
+subdir('feebas')
+subdir('milotic')
+subdir('castform')
+subdir('kecleon')
+subdir('shuppet')
+subdir('banette')
+subdir('duskull')
+subdir('dusclops')
+subdir('tropius')
+subdir('chimecho')
+subdir('absol')
+subdir('wynaut')
+subdir('snorunt')
+subdir('glalie')
+subdir('spheal')
+subdir('sealeo')
+subdir('walrein')
+subdir('clamperl')
+subdir('huntail')
+subdir('gorebyss')
+subdir('relicanth')
+subdir('luvdisc')
+subdir('bagon')
+subdir('shelgon')
+subdir('salamence')
+subdir('beldum')
+subdir('metang')
+subdir('metagross')
+subdir('regirock')
+subdir('regice')
+subdir('registeel')
+subdir('latias')
+subdir('latios')
+subdir('kyogre')
+subdir('groudon')
+subdir('rayquaza')
+subdir('jirachi')
+subdir('deoxys')
+subdir('turtwig')
+subdir('grotle')
+subdir('torterra')
+subdir('chimchar')
+subdir('monferno')
+subdir('infernape')
+subdir('piplup')
+subdir('prinplup')
+subdir('empoleon')
+subdir('starly')
+subdir('staravia')
+subdir('staraptor')
+subdir('bidoof')
+subdir('bibarel')
+subdir('kricketot')
+subdir('kricketune')
+subdir('shinx')
+subdir('luxio')
+subdir('luxray')
+subdir('budew')
+subdir('roserade')
+subdir('cranidos')
+subdir('rampardos')
+subdir('shieldon')
+subdir('bastiodon')
+subdir('burmy')
+subdir('wormadam')
+subdir('mothim')
+subdir('combee')
+subdir('vespiquen')
+subdir('pachirisu')
+subdir('buizel')
+subdir('floatzel')
+subdir('cherubi')
+subdir('cherrim')
+subdir('shellos')
+subdir('gastrodon')
+subdir('ambipom')
+subdir('drifloon')
+subdir('drifblim')
+subdir('buneary')
+subdir('lopunny')
+subdir('mismagius')
+subdir('honchkrow')
+subdir('glameow')
+subdir('purugly')
+subdir('chingling')
+subdir('stunky')
+subdir('skuntank')
+subdir('bronzor')
+subdir('bronzong')
+subdir('bonsly')
+subdir('mime_jr')
+subdir('happiny')
+subdir('chatot')
+subdir('spiritomb')
+subdir('gible')
+subdir('gabite')
+subdir('garchomp')
+subdir('munchlax')
+subdir('riolu')
+subdir('lucario')
+subdir('hippopotas')
+subdir('hippowdon')
+subdir('skorupi')
+subdir('drapion')
+subdir('croagunk')
+subdir('toxicroak')
+subdir('carnivine')
+subdir('finneon')
+subdir('lumineon')
+subdir('mantyke')
+subdir('snover')
+subdir('abomasnow')
+subdir('weavile')
+subdir('magnezone')
+subdir('lickilicky')
+subdir('rhyperior')
+subdir('tangrowth')
+subdir('electivire')
+subdir('magmortar')
+subdir('togekiss')
+subdir('yanmega')
+subdir('leafeon')
+subdir('glaceon')
+subdir('gliscor')
+subdir('mamoswine')
+subdir('porygon_z')
+subdir('gallade')
+subdir('probopass')
+subdir('dusknoir')
+subdir('froslass')
+subdir('rotom')
+subdir('uxie')
+subdir('mesprit')
+subdir('azelf')
+subdir('dialga')
+subdir('palkia')
+subdir('heatran')
+subdir('regigigas')
+subdir('giratina')
+subdir('cresselia')
+subdir('phione')
+subdir('manaphy')
+subdir('darkrai')
+subdir('shaymin')
+subdir('arceus')
+subdir('494')  # Pokemon Egg
+subdir('495')  # Bad Egg TODO verify this
```

Comment:
From experience, it's the bad egg.

Reply:
Is there are a reason we can't just name those folders egg and bad_egg?